students/smh2278.md

```diff
@@ -4,18 +4,16 @@ Name: Sebastian Hereu
 
 UNI: smh2278
 
-Github: [link](https://github.com/sebastianhereu)
+|  |  | 
+|:--|:--|
+|Name|Sebastian Hereu|
+|UNI| smh2278|
+| GitHub Profile | [link](https://github.com/sebastianhereu) |
+| GitHub README | [link](https://github.com/sebastianhereu/sebastianhereu/blob/main/README.md) |
+| project link (fundy) | [link](https://github.com/sebastianhereu/fundy) |
 
 ## Project Proposal
-
-Idea:
-
-1) Fundamental Analysis library - takes time series data and provides indicators like RSI and Kaufman Efficiency Ratio. Can also trigger trades to a
-specified API with certain events i.e. RSI crosses some value
+- [fundy](./projects/python/fundy.md)
```

Comment:
update link path and re-squash 😈

Reply:
Is my solution fine?

I made my fixes, committed them, and then ran 

git reset --soft "HEAD^"
git commit --amend

to change history and ammend with the original commit from rebasing.